### PR TITLE
Update shared-line-appearance.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@srividhc, @LolaJacobsen 

updated per feedback received from the field. Phone System licensing is required to enable this feature not just the EV enabled flag. 

This page, https://docs.microsoft.com/en-us/microsoftteams/here-s-what-you-get-with-phone-system, which is linked to this information is actually missing delgation, Group pickup, SLA :).

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/shared-line-appearance.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Shared line appearance is part of the delegation feature that lets a user choose
 
 A user must be an enterprise voice user to be a delegate or set up delegation and enable others to make or receive calls on their behalf.
 
-Both managers and delegates need to be enterprise voice enabled. The shared line experience is part of delegation and requires no additional license. For additional details on the licensing model, See [Office 365 licensing for Microsoft Teams](office-365-licensing.md).
+Both managers and delegates need to be enterprise voice enabled. The shared line experience is part of delegation and is included with Phone System. For additional details on the licensing model, See [Office 365 licensing for Microsoft Teams](office-365-licensing.md).
```
Comment (JohanFreelancer9) [inline Teams/shared-line-appearance.md]:
```suggestion
Both managers and delegates need to be Enterprise voice enabled. The shared line experience is part of delegation and is included with Phone System. For additional details on the licensing model, See [Office 365 licensing for Microsoft Teams](office-365-licensing.md).
```

Comment (carolynblanding) [reply Teams/shared-line-appearance.md]: Further information about this. Managers and delegates need to have PSTN calling capabilities which allows for Delegation and Dial Pad to show in the clients. Therefore, they either need Calling Plan license assigned or they need an Online Voice Routing Policy assigned. If there is no PSTN calling capability, Delegation will not be enabled. In fact call delegation requires both PSTN Calling capability and that call delegation is allowed by the Teams Calling Policy ( which it is by default). 

See the evaluation for this capability in the Teams diagnostics

callingSupportService: Calculated isDelegationAllowed with the following flags: {"isPSTNCallingAllowed":true,"isAllowDelegationPolicyHonored":true,"isDelegationAllowed":true,"result":true}. Old state: true, Current state true

@CarolynRowe  Didn't we decide to not use Enterprise Voice in our topics?

**Comment by CarolynRowe** [comment]
I made changes directly in the article so I'm going to close this pull request.  Changes should go live tomorrow.

**Pull request closed without merging**